### DISCUSSION TITLE
Plugin to show release notes

### DISCUSSION THREAD

**Discussion by tramdeholl** [composer/composer#9824] [Q&A / Support]
Hi There,
I'm working on a plugin to show release notes when using the update command with the `--dry-run` option.
I would like to add an option such as `--release-notes` to the update command but after some testing, it doesn't seem like this is possible using the plugin interface. I have a few questions below.

1. Would such a feature be better as a plugin using a separate command or as a PR using above option? It seems to be a better fit as an update option.
2. Am I allowed to write after the funding output? The plugin events I found only seem to allow writes above and this info I think would be better suited at the bottom.

**Comment by Seldaek** [comment]
IMO this would be good to start as a plugin, which can define its own command to begin with perhaps? If it pans out and sees some usage maybe we can eventually consider merging it in as a native Composer feature.
